edb/lib/cal.edgeql

```diff
@@ -292,7 +292,9 @@ std::`-` (l: cal::local_datetime, r: std::duration) -> cal::local_datetime {
 CREATE INFIX OPERATOR
 std::`-` (l: cal::local_datetime, r: cal::local_datetime) -> std::duration {
```

Comment:
We don't seem to have tests for `local_datetime` and `datetime` subtraction.  Please add a couple.

Reply:
Just find out that we need to fix python bindings for the test.

Reply:
Done